src/main/java/frc/trigon/robot/subsystems/climber/ClimberConstants.java

```diff
@@ -0,0 +1,142 @@
+package frc.trigon.robot.subsystems.climber;
+
+import com.ctre.phoenix6.configs.TalonFXConfiguration;
+import com.ctre.phoenix6.controls.Follower;
+import com.ctre.phoenix6.signals.InvertedValue;
+import com.ctre.phoenix6.signals.NeutralModeValue;
+import edu.wpi.first.math.geometry.Pose3d;
+import edu.wpi.first.math.geometry.Rotation3d;
+import edu.wpi.first.math.system.plant.DCMotor;
+import edu.wpi.first.units.Units;
+import edu.wpi.first.wpilibj.util.Color;
+import edu.wpi.first.wpilibj.util.Color8Bit;
+import edu.wpi.first.wpilibj2.command.sysid.SysIdRoutine;
+import frc.trigon.robot.constants.RobotConstants;
+import org.trigon.hardware.phoenix6.talonfx.TalonFXMotor;
+import org.trigon.hardware.phoenix6.talonfx.TalonFXSignal;
+import org.trigon.hardware.simulation.ElevatorSimulation;
+import org.trigon.utilities.mechanisms.ElevatorMechanism2d;
+
+public class ClimberConstants {
+    private static final int
+            MASTER_MOTOR_ID = 1,
+            FOLLOWER_MOTOR_ID = 12;
+    private static final String
+            MASTER_MOTOR_NAME = "ClimberMasterMotor",
+            FOLLOWER_MOTOR_NAME = "ClimberFollowerMotor";
+
+    static final TalonFXMotor
+            MASTER_MOTOR = new TalonFXMotor(
+            MASTER_MOTOR_ID,
+            MASTER_MOTOR_NAME,
+            RobotConstants.CANIVORE_NAME
+    ),
+            FOLLOWER_MOTOR = new TalonFXMotor(
+                    FOLLOWER_MOTOR_ID,
+                    FOLLOWER_MOTOR_NAME,
+                    RobotConstants.CANIVORE_NAME
+            );
+
```

Comment:
There doesn't need to be a newline here. Up to you just be consistent

Reply:
didn't we say we're having a blank line between the motor declaration and motor settings?

Reply:
I was told it's up to me. You can do it however you like